apps/dashboard/src/components/subscribers/create-subscriber-form.tsx

```diff
@@ -38,9 +39,9 @@ type CreateSubscriberFormProps = {
 };
 
 export const CreateSubscriberForm = (props: CreateSubscriberFormProps) => {
-  const track = useTelemetry();
   const { onSuccess } = props;
-
+  const track = useTelemetry();
+  const navigateToSubscribersFirstPage = useNavigateToSubscribersFirstPage();
```

Comment:
When the user creates a subscriber, then we want to navigate him to the first page